### DISCUSSION TITLE
Allow cycling of add to queue position

### DISCUSSION THREAD

**Discussion by looowizz** [sayaka-sh/spmp#317] [Ideas]
Hi! Absolutely love the feature to add songs to specific places in the queue.

I think it'd be great if the position cycled to the end of the queue when clicking the "-" button at the beginning of the queue and back to the start when clicking the "+" at the end.

i.e. clicking the circled button would change the position to add the song to the end of the queue
![The add to queue screen's "-" button circled and currently the position the song will be added to is "Play after next song"](https://github.com/toasterofbread/spmp/assets/47942583/caa219a6-29b0-4fa9-91c0-dc87a2e78363)

This would be really useful when wanting to add a song to the end of the queue as we don't have to spam the "+" until we reach the end of the queue :)

**Comment by sayaka-sh** [comment]
Long-pressing the button should jump to the start/end of the queue

**Reply by looowizz** [reply]
OHH thank you so much!
